worker/jobTypes/S3Publish.js

```diff
@@ -0,0 +1,52 @@
+const fs = require('fs-extra');
+const workerUtils = require('../utils/utils');
+
+class S3PublishClass {
+
+  constructor(currentJob) {
+    this.currentJob = currentJob;
+  }
+
+  async pushToStage(logger) {
+    const currentJob = this.currentJob;
+    logger.save(`${'(stage)'.padEnd(15)}Pushing to staging`);
+    try {
+      const exec = workerUtils.getExecPromise();
+      const command = `
+        . /venv/bin/activate && 
+        cd repos/${workerUtils.getRepoDirName(currentJob)} && 
```

Comment:
This getRepoDirName function should probably move into the GitHubJob class, then reference it here. I don't like that repos is hardcoded all over the place. So I'm wondering if we should put that in the GitHubJob class as well and only reference it there.